src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -417,8 +415,10 @@ private static boolean shouldBeAloneOnLineWithNotAloneOption(Details details,
     private static boolean isAloneOnLine(Details details, String targetSrcLine) {
         final DetailAST rcurly = details.rcurly;
         final DetailAST nextToken = details.nextToken;
-        return (!TokenUtil.areOnSameLine(rcurly, nextToken) || skipDoubleBraceInstInit(details))
-                && CommonUtil.hasWhitespaceBefore(details.rcurly.getColumnNo(), targetSrcLine);
+        return (nextToken == null || !TokenUtil.areOnSameLine(rcurly, nextToken)
+            || skipDoubleBraceInstInit(details))
+            && CommonUtil.hasWhitespaceBefore(details.rcurly.getColumnNo(),
+            targetSrcLine);
```

Comment:
Like previously mentioned, this line should be indented.

Reply:
Done